src/ui/pages/AssetInfo/AssetGlobalStats.tsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import { useCurrency } from 'src/modules/currency/useCurrency';
+import type { AssetFullInfo } from 'src/modules/zerion-api/requests/asset-get-fungible-full-info';
+import { formatCurrencyValue } from 'src/shared/units/formatCurrencyValue';
+import { formatPercent } from 'src/shared/units/formatPercent/formatPercent';
+import { HStack } from 'src/ui/ui-kit/HStack';
+import { UIText } from 'src/ui/ui-kit/UIText';
+
+const ONE_DAY = 24 * 60 * 60 * 1000;
+const ONE_HOUR = 60 * 60 * 1000;
+
+function AssetStatsChip({
+  title,
+  fullTitle,
+  value,
+}: {
+  title: string;
+  fullTitle?: string;
+  value: string;
+}) {
+  return (
+    <HStack
+      gap={4}
+      style={{
+        backgroundColor: 'var(--neutral-200)',
+        padding: '6px 8px',
+        borderRadius: 8,
+      }}
+      title={fullTitle}
+    >
+      <UIText kind="caption/accent" color="var(--neutral-500)">
+        {title}
+      </UIText>
+      <UIText kind="caption/accent">{value}</UIText>
+    </HStack>
+  );
+}
+
+const numberFormatter = Intl.NumberFormat('en', {
+  notation: 'compact',
+  maximumFractionDigits: 1,
+});
+
+export function AssetGlobalStats({
+  assetFullInfo,
+}: {
+  assetFullInfo: AssetFullInfo;
+}) {
+  const { currency } = useCurrency();
+
+  const createdRecently =
+    Date.now() - new Date(assetFullInfo.extra.createdAt).getTime() < ONE_DAY;
+  const ageInHours = Math.floor(
+    (Date.now() - new Date(assetFullInfo.extra.createdAt).getTime()) / ONE_HOUR
+  );
+
+  return (
+    <HStack gap={8} style={{ width: '100%', overflowX: 'auto' }}>
+      {createdRecently ? (
+        <AssetStatsChip title="AGE" value={`${ageInHours}h`} />
+      ) : null}
+      {assetFullInfo.fungible.meta.fullyDilutedValuation != null ? (
+        <AssetStatsChip
+          title="FDV"
+          fullTitle="Fully Diluted Valuation"
+          value={formatCurrencyValue(
+            assetFullInfo.fungible.meta.fullyDilutedValuation,
+            'en',
+            currency,
+            {
```

Comment:
The formatting options are repeated multiple times in this file. We could consider extracting these options into a constant